test/cbmc/proofs/OTA_ActivateNewImage/OTA_ActivateNewImage_harness.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * AWS IoT Over-the-air Update v3.1.0
+ * Copyright (C) 2021 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file OTA_ActivateNewImage_harness.c
+ * @brief Implements the proof harness for OTA_ActivateNewImage function.
+ */
+/*  Ota Agent includes. */
+#include "ota.h"
+#include "otaAgentStubs.c"
+
+extern OtaAgentContext_t otaAgent;
+
+void OTA_ActivateNewImage_harness()
+{
+    OtaErr_t err;
+    OtaInterfaces_t otaInterface;
+
+    otaInterface.pal.activate = palActivate;
+
+    otaAgent.pOtaInterface = &otaInterface;
+
+    /* otaAgent.pOtaInterface is checked in OTA_Init to be non-NULL. */
+    __CPROVER_assume( otaAgent.pOtaInterface != NULL );
```

Comment:
I do not think that we need this assumption since we are assigning the pointer with address of a variable in line 40.